deployments/cloudfoundry/bosh/release

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+# This script is used to create a Bosh release of the Splunk distribution of the OpenTelemetry Collector.
+# This is intended to be run by the Pivotal Cloud Foundry (PCF) tile. The PCF tile should specify
+# proper variables as described below.
+
+# Variables:
```

Comment:
Should these env vars be documented in the dev guide and/or readme?

Reply:
It's probably good to have a reference to them, but I don't want the two places to get out of sync. I'll add a comment where the release script is used in the REAMDE to reference the script itself for more variables.